core/src/main/scala/org/apache/spark/util/taskListeners.scala

```diff
@@ -55,14 +55,16 @@ class TaskCompletionListenerException(
   extends RuntimeException {
 
   override def getMessage: String = {
-    if (errorMessages.size == 1) {
```

Comment:
Aside: Was just curious about the naming - interesting. Is this common pattern in spark code ?